risks/MASVS-STORAGE/1-store-sensitive-data-securely/data-unencrypted-external/android-data-unencrypted-external-static/test.md

```diff
@@ -0,0 +1,25 @@
+---
+platform: android
+title: Usage of API to get to external storage locations
+apis: [Environment#getExternalStoragePublicDirectory, Environment#getExternalStorageDirectory, Environment#getExternalFilesDir, Environment#getExternalCacheDir]
+type: [static]
+---
+
+## Overview
+
+Developers can obtain paths to the external storage locations with `Environment.getExternal...` methods. This test searches for a set of APIs that are commonly used for obtaining external storage locations in source code.
+
+## Steps
+
+1. Run a [static analysis](../../../../../techniques/android/MASTG-TECH-0014.md) tool on the app and look for uses of `getExternal...` API.
```

Comment:
I added a description and links to the `test.md` but I kept only `WRITE_EXTERNAL_STORAGE`, `MANAGE_EXTERNAL_STORAGE`, `requestLegacyExternalStorage`, `preserveLegacyExternalStorage` for now. I need to read more about the rest because they are still not clear to me.